CMakeLists.txt

```diff
@@ -177,6 +178,13 @@ add_library(lego1 SHARED
   LEGO1/viewmanager.cpp
 )
 
+if (ISLE_USE_SMARTHEAP)
+  target_compile_definitions(lego1 PRIVATE USE_SMARTHEAP)
```

Comment:
```suggestion
  target_compile_definitions(lego1 PRIVATE USE_SMARTHEAP)
  target_compile_options(lego1 PRIVATE "/FI${CMAKE_SOURCE_DIR}/3rdparty/smartheap/SMRTHEAP.HPP")
```

Reply:
Good idea. I was including `SMRTHEAP.HPP` through the `USE_SMARTHEAP` macro (currently just `isleapp.cpp`), but this is probably cleaner/more reliable.